src/utils/audio-manager.js

```diff
@@ -0,0 +1,43 @@
+import { func } from 'prop-types';
+
+class AudioManager {
+  constructor(url) {
+    if (!url) {
+      throw new Error('You have to pass a url as the argument!');
+    }
+    this.audio = new Audio(url);
+  }
+
+  play() {
+    this.audio
```

Comment:
Since the `url` above is not mandatory, you can handle a check here and just `return;` if `this.audio` has no `src` attribute